_pages/en/strategy-build-vs-buy.md

```diff
@@ -37,66 +44,140 @@ th {
+      - IT Solutions Architecture is a key partner throughout the lifecycle of the IT Solutions but IT Products teams have authority to operate and take responsibility for their own products.
+      - IT Solutions are architected iteratively over time, with each gate and iteration providing additional information as to choosing the best IT Products to compose them (see [Target Solution Delivery Model](strategy-target-solution-delivery-model.html)).
+      - Leveraging external third party integrators is supported but IT Solutions Architecture remains in control to ESDC throughout its lifecycle and abides by its [Architectural Principles](#architecture).
+   2. Complex but not core to mandate IT Solutions are designed internally.
+   3. Non-complex and not core to mandate IT Solutions may be designed and delivered by third party integrators.
+3. IT Solutions, whether built internally or built by a third party and delivered as a set of pre-packaged COTS to configure and/or as software as a service, adhere to the Architectural principles as listed in section [Architecture](#architecture)
+4. IT Solutions must provide their overall and individual IT products metrics internally to ESDC. Metrics are comprised of the following:
+   - Accessibility test results
+   - Integration testing results
+   - Security test results
+   - API consumption statistics
+   - Production deployment audit traces
+   - Availability statistics
+5. Security Assessment & Authority (SA&A) uses the Target-SA&A methodology
+6. Accessibility Assessment uses the Target-Accessibility methodology
+7. Audit Assessment uses the Target-Audit methodology
+8. IT Solutions reuse existing IT Products, including:
+    - Internally developed software
+    - Open Source Software (internally and externally developed)
+    - Third party Services (SaaS, PaaS, and IaaS) and COTS
+
+### Finance
+
+1. IT Solutions sizes are minimized as much as possible
+2. IT Solutions leverage existing, reusable, scalable and approved IT Products to minimize cost when possible
+   - IT Products requiring purchased licences are not deemed as reusable or scalable for the purpose of this criterion.
+3. IT Solutions, once in production, have dedicated and recurring budgets to support their evolution over their lifecycle, and are managed in tandem by the primary business owner and the dedicated IT Products teams.
+   - The IT Products teams are responsible for maintaining their own products as part of the IT Solutions. (see [Target Solution Delivery Model](strategy-target-solution-delivery-model.html))
+   - A same team managing an IT Product may provide service to multiple IT Solutions; IT Products are thus required to have a scalable and easily adaptable financial model supporting multiple business owners if required (similar to SaaS pay as you use or yearly subscriptions per users, etc.).
```

Comment:
Right, need to work on that definition then as IT Products may be reused for many IT Solutions. This in turn supports the TOM proposed since enabling teams would provide access to reusable and scalable IT Products that may or may not be used by certain IT Solutions.

Reply:
I've actually removed the IT Architecture section... Did you have any specific rules you would see here though?